### ISSUE TITLE
Parameter reference (%1) could not be found

### ISSUE THREAD

**Issue by ryankmcintyre** [Microsoft/mssql-docker#204]
Running Docker on Windows 10, I pulled the latest mssql-server-windows-developer image. Executed:
`docker run -d -p 1433:1433 --env sa_password=Password1 --env ACCEPT_EULA=Y --isolation=hyperv microsoft/mssql-server-windows-developer`
Resulted in the error:
"docker: Error response from daemon: container 05a9878069205b35f53e10f04e641733aeacfedb9ab05288b1b7621e4112fa8b encountered an error during CreateContainer: failure in a Windows system call: The description string for parameter reference (%1) could not be found."

**Comment by andrewNestorenko** [comment]
the same here, but with all containers

**Comment by ryankmcintyre** [comment]
@andrewNestorenko You're right, it's all containers on Windows. I've been in Linux mode for a while so didn't notice until I switched over. Found [this](https://github.com/docker/for-win/issues/1255), but still looks to be an open issue.

**Comment by yrest** [comment]
@ryankmcintyre , which version of docker and build of W10 is it on?

**Comment by ryankmcintyre** [comment]
I'm on Windows 17025. Docker version is more interesting in that when I look at "About Docker" from the Windows app it shows 17.09.0-ce-win33, but when I run "docker --version" it shows 17.04.0-ce.

**Comment by yrest** [comment]
Thanks Ryan! My current is 17.11.0-ce-win40 (14328), but it makes no difference whatsoever on WIP.
